scripts/actions/mobskills/stygian_vapor.lua

```diff
@@ -10,7 +10,13 @@ mobskillObject.onMobSkillCheck = function(target, mob, skill)
 end
 
 mobskillObject.onMobWeaponSkill = function(target, mob, skill)
-    skill:setMsg(xi.mobskills.mobStatusEffectMove(mob, target, xi.effect.PLAGUE, 5, 0, 60))
+    local typeEffect = xi.mobskills.mobStatusEffectMove(mob, target, xi.effect.PLAGUE, 3, 0, 120)
```

Comment:
I wrote a really long post here about how to fix this to match jimmys sheet then i actually calculated what it would do... lmao

power should be 5 because (per plague effect lua)

```lua
    target:addMod(xi.mod.REFRESH_DOWN, math.ceil(effect:getPower() / 2))
    target:addMod(xi.mod.REGAIN_DOWN, effect:getPower() * 10)
```
math.ceil(5/2) = 3
5*10 = 50

math.ceil is basically the opposite of floor, it rounds up
math.ceil(2) = 2, but math.ceil(2.00001) = 3

Reply:
Oops, we definitely went through this before. Got that adjusted.